http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -787,8 +813,12 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       raw"""galaxy.rbac.enabled=false""",
       raw"""galaxy.rbac.serviceAccount=${ksa.value}""",
       raw"""rbac.serviceAccount=${ksa.value}""",
-      // TODO Update during https://broadworkbench.atlassian.net/browse/IA-2171
-      raw"""persistence={}"""
+      raw"""persistence.nfs.persistentVolume.extraSpec.gcePersistentDisk.pdName=${nfsDisk.name.value}""",
+      raw"""persistence.nfs.size=${nfsDisk.size.gb.toString}""",
+      raw"""persistence.postgres.persistentVolume.extraSpec.gcePersistentDisk.pdName=${appName.value}-${config.galaxyAppConfig.postgresDiskNameSuffix}""",
+      raw"""persistence.postgres.size=${config.galaxyAppConfig.postgresDiskSize.gb.toString}""",
+      raw"""nfs.persistence.existingClaim=nfs-disk-pvc""",
+      raw"""galaxy.postgresql.persistence.existingClaim=postgres-disk-pvc"""
```

Comment:
Based on [Alex's comment](https://github.com/galaxyproject/galaxykubeman-helm/pull/7#issuecomment-689904354) from earlier, I wonder if we should set these two names ourselves instead of using Galaxy chart's defaults so if they happen to change the default, we wouldn't be affected; e.g. 

```
--set persistence.nfs.name=custom-nfs-name
--set persistence.postgres.name=custom-postgres-name
```

and then

```
--set nfs.persistence.existingClaim=custom-nfs-name-pvc
--set galaxy.postgresql.persistence.existingClaim=custom-postgres-name-pvc
```

Reply:
I was thinking the same and even had the question of - can two PVCs in the same cluster have the same name? with the current implementation (ie using the values that exist in the chart), there would be multiple PVCs with the same name and I don't think that's ideal or even correct. Going to be reaching out to Alex for advice on this

Reply:
PVC is a namespaced resource so I don't think multiple PVCs using the same name will be a problem as long as they belong to distinct namespaces. To clarify, my comment was about not depending on the Galaxy default PVC names.

Reply:
ah i see, that makes sense - will update!